tests/e2e/default/enketo/training-cards.wdio-spec.js

```diff
@@ -58,6 +59,51 @@ describe('Training Cards', () => {
     expect(await reportsPage.leftPanelSelectors.allReports()).to.be.empty;
   });
 
+  it('should display confirm message before navigating to different page', async () => {
+    await commonPage.goToMessages();
+    await commonElements.waitForPageLoaded();
+    await setLastViewedDateInThePast();
+    // Unfinished trainings should appear again after reload.
+    await browser.refresh();
+    await trainingCardsPage.waitForTrainingCards();
+    await commonPage.goToPeople();
+
+    const confirmMessage = await modalPage.getModalDetails();
+    expect(confirmMessage.header).to.contain(expectedConfirmModalHeader);
+    expect(confirmMessage.body).to.contain(expectedConfirmMessage);
+    expect(await browser.getUrl()).to.contain('/messages');
+
+    await trainingCardsPage.confirmQuitTraining();
+    await trainingCardsPage.checkTrainingCardIsNotDisplayed();
+    expect(await browser.getUrl()).to.contain('/contacts');
+
+    await commonPage.goToReports();
```

Comment:
I think we need a clarifying comment for this. It wasn't straight forward for me at first
```suggestion
    // Checking the training wasn't saved in Reports page
    await commonPage.goToReports();
```